DxDispatch/src/dxdispatch/Adapter.cpp

```diff
@@ -73,21 +68,7 @@ std::vector<Adapter> Adapter::GetAll()
     std::vector<Adapter> adapters;
 
 #ifdef _GAMING_XBOX
-    Microsoft::WRL::ComPtr<ID3D12Device> device;
```

Comment:
>Microsoft::WRL::ComPtr<ID3D12Device> device; [](http://example.com/codeflow?start=4&length=44)

No longer an interesting target? 🤔

Reply:
Nah, it's just that there's no good reason to enumerate here (only one GPU is present) and it was causing some issues on process exit.